sonic_sfp/sff8436.py

```diff
@@ -923,6 +972,66 @@ def calc_rx_power(self, eeprom_data, offset, size):
              'type': 'nested',
              'decode': dom_channel_monitor_values}}
 
+# new added parser for some specific values interested by SNMP
+# TO DO: find a way to reuse the definitions in above code, need refactor
+    dom_module_temperature = {
+        'Temperature':
+            {'offset':0,
+             'size':2,
+             'type': 'func',
+             'decode': {'func':calc_temperature}}
+        }
+
+    dom_module_voltage = {
+        'Vcc':
+            {'offset':0,
+             'size':2,
+             'type': 'func',
+             'decode': { 'func':calc_voltage}}
```

Comment:
Delete space after `{` for consistent PEP8 style (`'decode': {'func':calc_voltage}}`).

There are many more occurrences. Please do this throughout the entire file.